torch/ao/quantization/fx/prepare.py

```diff
@@ -125,7 +128,7 @@ def node_arg_is_bias(node: Node, arg: Any) -> bool:
 def is_input_arg_dtype_supported_by_backend(
     arg: Argument,
     node: Node,
-    node_name_to_target_dtype: Dict[str, Dict[str, Optional[torch.dtype]]],
+    node_name_to_target_dtype: Dict[str, Dict[str, Optional[Union[torch.dtype, type]]]],
```

Comment:
I feel we probably want to define a type that can represent both Tensor (with dtypes) and primitive types so that they can be easily identified (maybe later in a separate PR)

Reply:
Yeah getting the type hints to work for this was a bit of a pain, that's a good idea